README.adoc

```diff
@@ -3,10 +3,83 @@
 
 == Introduction
 
-The Jakarta Data specification provides an API that allows easy access to database technologies. A
-Java developer can split the persistence from the model with several features,
-such as the ability to compose custom query methods on a `Repository` interface where the framework
-will implement it.
+The Jakarta Data specification provides an API for easier data access. A Java developer can split the persistence from the model with several features, such as the ability to compose custom query methods on a Repository interface.
+
+Jakarta Data’s goal is to provide a familiar and consistent, Jakarta-based programming model for data access while still retaining the particular traits of the underlying data store.
+
+=== Repository
+
+A repository abstraction aims to significantly reduce the boilerplate code required to implement data access layers for various persistence stores.
+
+[source,java]
+----
+@Repository
+public interface CarRepository extends CrudRepository<Car, Long> {
+
+  List<Car> findByType(Type type);
```

Comment:
Avoid including the`Type` class in the example if we don't define what it is.  Users will wonder if it comes from the specification and where to find it.  An easy change would be to switch it to `String`.